config/development.json

```diff
@@ -4,10 +4,7 @@
 		"max_static_paths": 1
 	},
 	"dev_dot": {
-		"max_static_paths": 1,
-		"algolia": {
-			"tutorialsIndexName": "staging_LEARN"
-		}
```

Comment:
Removes "staging" `algolia` settings from `development.json`, so that we're always seeing data from the production index.

This is a bit of an adjacent change, rationale is that we'll _usually_ want to see our production search index when running the site locally, and will likely only every want to switch to `staging_DEVDOT_omni` if we're making search-related changes.

Reply:
💭 Makes me wonder if we have docs related to our search indices with this info. Like specifically doing local search development on the staging index.

Reply:
ooh yeah these would be a great addition 👍 made a task for myself to add some README notes in https://github.com/hashicorp/dev-portal/blob/main/docs/README.md : 🎟 Add notes on Algolia config to DevDot README